.dockerignore

```diff
@@ -0,0 +1,12 @@
+#ignore git history to avoid commiting sensitive info
+.git
+# ignore Dockerfile
+Dockerfile
```

Comment:
@securityclippy We provide Docker as a supported use case. We should include the Dockerfile as part of the project.